quesma/frontend_connectors/elasticsearch_ingest.go

```diff
@@ -0,0 +1,137 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
+package frontend_connectors
+
+import (
+	"context"
+	"fmt"
+	"github.com/ucarion/urlpath"
+	"net/http"
+	quesma_api "quesma_v2/core"
+)
+
+type ElasticsearchIngestFrontendConnector struct {
+	BasicHTTPFrontendConnector
+}
+
+const (
+	IndexDocPath  = "/:index/_doc"
+	IndexBulkPath = "/:index/_bulk"
+
+	// IngestAction and below are metadata items passed to processor.
+	IngestAction    = "ingest_action"
+	DocIndexAction  = "_doc"
+	BulkIndexAction = "_bulk"
+	IngestTargetKey = "ingest_target"
+	// TODO: this actually should not be a dependency on processor
+)
+
+func NewElasticsearchIngestFrontendConnector(endpoint string) *ElasticsearchIngestFrontendConnector {
+	fc := &ElasticsearchIngestFrontendConnector{
+		BasicHTTPFrontendConnector: BasicHTTPFrontendConnector{
```

Comment:
As we are embedding `BasicHTTPFrontendConnector` below methods should not be needed and can be removed

Reply:
https://github.com/QuesmaOrg/quesma/pull/1055/commits/e2ec36bbdc86eee719142405de97489a5061fa4d